Purchases/Caching/RCDeviceCache.h

```diff
@@ -14,31 +14,42 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface RCDeviceCache : NSObject
 
+- (instancetype)initWith:(NSUserDefaults *)userDefaults;
```

Comment:
this diff is probably easier to read in split view. 
it's mostly separating the stuff for offerings and purchaser info, and shuffling stuff around so it's easier to read.